nucleus/modelci/unit_test.py

```diff
@@ -119,4 +117,43 @@ def get_eval_history(self) -> List[UnitTestEvaluation]:
         Returns:
             A list of UnitTestEvaluation objects.
         """
-        return self._client.get_unit_test_eval_history(self.id)
+        response = self._client._connection.get(
+            f"modelci/unit_test/{self.id}/eval_history",
+        )
+        # TODO(gunnar): Repeated info calls are slow -> Move work to backend
+        eval_history = GetEvalHistory.parse_obj(response)
+        return [
+            self.get_unit_test_eval_info(evaluation.id)
+            for evaluation in eval_history.evaluations
+        ]
+
+    def info(self):
+        """Retrieves info of the Unit Test.
+
+        Args:
+            unit_test_id: ID of Unit Test
+
+        Returns:
+            A UnitTestInfo object
+        """
+        response = self._client._connection.get(
+            f"modelci/unit_test/{self.id}/info",
+        )
+        return UnitTestInfo(**response)
+
+    def get_unit_test_eval_info(
```

Comment:
Moved from ModelCI `client`

Reply:
I don't think this method is necessary or belongs in `unit_test.py` now.